apps/web/screens/ServiceDetails/ServiceDetails.tsx

```diff
@@ -55,15 +62,46 @@ const ServiceDetails: Screen<ServiceDetailsProps> = ({
         <SidebarLayout sidebarContent={<></>}>
           <SubpageMainContent
             main={
-              !service ? (
-                <Box>
-                  <Text variant="h3" as="h3">
-                    {n('serviceNotFound')}
-                  </Text>
+              <Box>
+                <Box marginBottom={2}>
+                  <Link
+                    className={classNames(styles.showMobile)}
+                    href={serviceListLink}
+                  >
+                    <Button
+                      colorScheme="default"
+                      iconType="filled"
+                      preTextIcon="arrowBack"
+                      preTextIconType="filled"
+                      size="small"
+                      type="button"
+                      variant="text"
+                    >
+                      {n('linkTextVefthjonustur')}
+                    </Button>
+                  </Link>
+                  <span className={classNames(styles.showDesktop)}>
```

Comment:
unncessary `classNames`, but this can also be achieved by `<Box display={['none', 'inline']}]>`. The current `showDesktop` className is also misleading because this is visible on everything above `breakpoints.sm` (576px) which would include all kinds of devices